src/components/itemContextMenu.js

```diff
@@ -616,10 +640,17 @@ function editItem(apiClient, item) {
 function deleteItem(apiClient, item) {
     return new Promise(function (resolve, reject) {
         import('../scripts/deleteHelper').then((deleteHelper) => {
-            deleteHelper.deleteItem({
-                item: item,
-                navigate: false
```

Comment:
In most cases I believe this caused two navigations.

Navigation was also being handled by the methods that called itemContextMenu.show().

Although I think this change might be better off in a different branch, I'm looking at. So I will revert this.

On that topic, scripts/deleteHelper.js is only used in itemContextMenu.js. Is this something we should consider moving entirely into itemContextMenu instead of having a separate script